source/help/reporting_issues.markdown

```diff
@@ -46,15 +46,24 @@ sensor:
 
 Make sure that you don't post your username, password, API key, access token or other [secrets](/docs/configuration/secrets/).
 
-### Traceback (if applicable)
+### Diagnostics information
 
-If things go wrong there will be a so-called traceback or an error message in other words in your log. Please include this. It starts with **Traceback** and can contain information where the error was triggered in the code.
+Consider uploading [the diagnostics file](docs/configuration/troubleshooting/#debug-logs-and-diagnostics) along with your issue report to allow faster triaging and pinpointing the issue.
+The information contained in the generated diagnostics file is redacted to avoid any sensitive information while still remaining useful for developers  to fix the issue.
```

Comment:
```suggestion
The information contained in the generated diagnostics file is redacted to avoid any sensitive information while still remaining useful for developers to fix the issue.
```